redfish-core/include/privileges.hpp

```diff
@@ -224,6 +224,13 @@ inline const Privileges& getUserPrivileges(const std::string& userRole)
         static Privileges readOnly{"Login", "ConfigureSelf"};
         return readOnly;
     }
+    if (userRole == "priv-oemibmserviceagent")
+    {
+        static Privileges admin{"Login", "ConfigureManager", "ConfigureSelf",
+                                "ConfigureUsers", "ConfigureComponents",
+                                "OemIBMPerformService"};
+        return admin;
```

Comment:
This doesn't look correct

Reply:
Can you be more specific?  I'll correct the variable name per the previous comment.  And the set of privileges is the same as admin, plus the OemIBMPerformService privilege.

Reply:
yeah sorry, i should have been more specific. 
Can you follow the pattern and call this something like ibmserviceagent